### PR TITLE
Avoid .babelrc conflict with React Native

### PR DESCRIPTION
**Pull request by aksonov**
React Native uses different babel settings

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "kefir",
-  "version": "3.2.2",
+  "version": "3.2.3",
```
Comment (rpominov) [inline package.json]: Could not change the version? The release script does that automatically.

**Comment by aksonov** [comment]
@rpominov done

**Comment by rpominov** [comment]
Thanks, I'll release a new version with this fix later today.

**Comment by rpominov** [comment]
@aksonov It's released.

**Comment by aksonov** [comment]
Thanks! I've just released https://github.com/aksonov/react-native-reactive based on Calmm-JS and KefirJS as Redux replacement.

**Pull request merged**